styles/main.css

```diff
@@ -187,7 +185,7 @@ div .back-to-top:hover {
 
 main {
     display: grid;
-    grid-template-columns: 350px 350px;
+    grid-template-columns: 1fr 1fr;

```

Comment:
While you do have two equally spaced columns, these columns are a fixed width.  Better to use a unit like fr to make the widths adaptable.